crates/metastore/src/errors.rs

```diff
@@ -87,6 +87,12 @@ pub enum MetastoreError {
 
     #[error(transparent)]
     Validation(#[from] sqlbuiltins::validation::ValidationError),
+
+    #[error("{0}")]
+    String(String),
```

Comment:
True, though I've seen this in various other error definitions in other crates so I figured this may be fine. Let me see if there's a another more refined alternative.

Reply:
For what it's worth, I've starting to come to the opinion that this is the easiest way to get a reasonable error. We currently have a lot of different error types and variants, and I originally approached it as "one failure case = one error variant", but this is tedious.

Also rust doesn't have backtraces for errors by default (and it's actually kinda painful to get them), and so if you use the same error variant twice in different spots, it becomes difficult to know which path caused that error.

Reply:
Yeah, this was my thinking too—too many error variants getting out of hand. 

Though in this case there is indeed a more suitable existing option for my use case that I missed (`MetastoreError::FailedInProcessStartup`), so I'll use that instead.